models/ai87net-mobilefacenet_112.py

```diff
@@ -98,8 +98,7 @@ def _create_bottleneck_stage(self, setting, bias, depthwise_bias,
 
     def forward(self, x):  # pylint: disable=arguments-differ
         """Forward prop"""
-        if x.shape[1] == 6:
-            x = x[:, 0:3, :, :]
+        x = x[:, 0:3, :, :]
```

Comment:
Check for the input input to have more than 3 channels.

Reply:
A check depending on the input shape will be a dynamic control. Due to the symbolic trace limitations, we can't add it straightforwardly. We will need additional code in train.py to initialize the model according to the input shape. I think someone who needs to use this model with less than 3 channels should modify the model, they can also change the forward.